assets/src/components/amp-support/index.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { Button, ExternalLink } from '@wordpress/components';
+import apiFetch from '@wordpress/api-fetch';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { Selectable } from '../selectable';
+import { AMPNotice } from '../amp-notice';
+import ClipboardButton from '../clipboard-button';
+import { SiteInfo } from './site-info';
+import { Theme } from './theme';
+import { Plugins } from './plugins';
+import { Errors } from './errors';
+import { ErrorSources } from './error-sources';
+import { ValidatedUrls } from './validated-urls';
+import { RawData } from './raw-data';
+
+/**
+ * AMP Support component.
+ *
+ * @class
+ * @param {Object} props Props for component.
+ * @return {JSX.Element} Makrup for AMP support component
+ */
+export function AMPSupport( props ) {
+	const { data, restEndpoint, args, nonce } = props;
+
+	const [ sending, setSending ] = useState( false );
+	const [ uuid, setUuid ] = useState( null );
+	const [ error, setError ] = useState( null );
+	const [ hasCopied, setHasCopied ] = useState( false );
+	const [ submitSupportRequest, setSubmitSupportRequest ] = useState( false );
+
+	/**
+	 * Event callback for send button.
+	 */
+	useEffect( () => {
+		( async () => {
+			if ( false === submitSupportRequest ) {
+				return;
+			}
+
+			setSending( true );
+			setUuid( null );
+			setError( null );
+
+			try {
+				apiFetch.use( apiFetch.createNonceMiddleware( nonce ) );
+
+				const response = await apiFetch(
+					{
+						path: restEndpoint,
+						method: 'POST',
+						data: {
+							args,
+						},
+					},
+				);
+
+				if ( undefined !== response.success && undefined !== response.data ) {
```

Comment:
Should we also check if the `uuid` property is present in the response object like:

```suggestion
				if ( undefined !== response.success && undefined !== response?.data?.uuid ) {
```